internal/webhook/spinapp_validating_test.go

```diff
@@ -33,45 +16,3 @@ func TestValidateReplicas(t *testing.T) {
 	fldErr = validateReplicas(spinv1alpha1.SpinAppSpec{Replicas: 1})
 	require.Nil(t, fldErr)
 }
-
-func TestValidateAnnotations(t *testing.T) {
```

Comment:
How come we're getting rid of these?

Reply:
They only make sense in the context of an executor right now - which when it's not _necessarily_ there already, would cause inconsistent validation